OptimoveSDK/Sources/Classes/Optimobile/Optimobile+Push.swift

```diff
@@ -209,27 +209,22 @@ extension Optimobile {
         Parameters:
             - notification: The notification which triggered the action
     */
-    static func pushTrackOpen(notification: PushNotification?) {
-        guard let notification = notification else {
+    static func pushTrackOpen(notification: PushNotification) {
+        if notification.id == 0 {
+            Logger.error("""
+            Failed to track the push open event.
+            Reason: Invalid notification id (== 0).
+            Payload: \(notification).
+            """)
             return
         }
-
         let params = ["type": KS_MESSAGE_TYPE_PUSH, "id": notification.id]
         Optimobile.trackEvent(eventType: OptimobileEvent.MESSAGE_OPENED, properties:params)
     }
 
-    @available(iOS 9.0, *)
-    internal func pushHandleOpen(withUserInfo: [AnyHashable: Any]?) {
```

Comment:
Is it safe to make these args non-optional?

Reply:
By default, this method is marked as internal and not exposed to the public. If someone tries to pass 'nil', the compiler will throw an error.

Reply:
Cool so long as everything changed is internal and it won't be breaking, seems fine.

Was just curious why we had them mostly as optional, if there's a path where we can call into it without a valid model or something.

Reply:
> Also, presume this method has no consumers left as we don't support iOS 9 now? (so no swizzles call to this from app-delegate methods?)

It is not in use in other parts of the code.